client/components/app.jsx

```diff
@@ -82,6 +83,13 @@ export default class App extends React.Component {
             addToCart={this.addToCart} />
         </div>
       );
+    } else if (viewName === 'cart') {
```

Comment:
Good work with this! You can also use a switch statement for this portion of the render as well.